spec/snowflake/client_spec.rb

```diff
@@ -112,6 +111,23 @@
       end
     end
 
+    context "with NUMBER and HighPrecision" do
+      # We have setup a simple table in our Snowflake account with the below structure:
+      # CREATE TABLE ruby_snowflake_client_testing.public.test_big_datatypes
+      #   (ID NUMBER(38,0), BIGFLOAT NUMBER(8,2));
+      # And inserted some test data:
+      # INSERT INTO test_big_datatypes VALUES (1, 8.2549);
```

Comment:
nitpick / double-checking: for the purpose of this test does it matter that the insert did `8.25XX`? the data got truncated in insertion, right? The only thing that applies to the test is the fact that the value is currently `8.25`?